code/modules/projectiles/guns/projectile/constructable/siegecannon.dm

```diff
@@ -253,7 +253,7 @@
 
 /obj/item/cannonball/iron/throw_impact(atom/hit_atom, var/speed, mob/user)
 	..()
-	if(isliving(hit_atom))
+	if(isliving(hit_atom) && cannonFired)
```

Comment:
boooooooooooooooooooooooooooooooooooooooooooooooooooo

Reply:
If you want your knockdown you will provide me dopamine by using my cannon. No more freebies!

Reply:
sneakycoded...

Reply:
I did mention it, just very vaguely